tests/e2e/enform.spec.js

```diff
@@ -0,0 +1,298 @@
+//const {test, expect} = require('@wordpress/e2e-test-utils-playwright');
+const {test, expect} = require('@playwright/test');
+
+import {rest} from './tools/lib/rest';
+import {login} from './tools/lib/login';
+
+import fieldsData from './fixtures/enform/ensapi_sample_fields_response.json'
+import questionsData from './fixtures/enform/ensapi_sample_questions_response.json'
+import questionData from './fixtures/enform/ensapi_sample_question_236734_response.json'
+import optinData from './fixtures/enform/ensapi_sample_question_3877_response.json'
+import depOptinData from './fixtures/enform/ensapi_sample_question_220954_response.json'
+import { formFields, formFieldsAttributes } from './fixtures/enform/enformData';
+
+test.describe.configure({ mode: 'serial' });
```

Comment:
I just saw in the [docs](https://playwright.dev/docs/test-parallel#serial-mode) that they don't recommend using `serial`, but maybe in this case it makes sense 🤔